packages/react-composites/src/composites/MeetingComposite/adapter/MeetingAdapter.ts

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/* eslint-disable @typescript-eslint/adjacent-overload-signatures */
+
+import {
+  CallAdapter,
+  CallIdChangedListener,
+  DisplayNameChangedListener,
+  IsMuteChangedListener,
+  IsScreenSharingOnChangedListener,
+  IsSpeakingChangedListener,
+  ParticipantJoinedListener,
+  ParticipantLeftListener
+} from '../../CallComposite';
+import { ChatAdapter, MessageReadListener, MessageReceivedListener, MessageSentListener } from '../../ChatComposite';
+import { MeetingState } from '../state/MeetingState';
+
+export type ConflictingProps = 'getState' | 'onStateChange' | 'offStateChange' | 'on' | 'off';
+
+export interface MeetingAdapter extends Omit<ChatAdapter, ConflictingProps>, Omit<CallAdapter, ConflictingProps> {
+  // Meeting-specific interfaces
+  getState(): MeetingState;
+  onStateChange(handler: (state: MeetingState) => void): void;
+  offStateChange(handler: (state: MeetingState) => void): void;
+
+  on(event: 'participantsJoined', listener: ParticipantJoinedListener): void;
+  on(event: 'participantsLeft', listener: ParticipantLeftListener): void;
+  on(event: 'meetingEnded', listener: ParticipantLeftListener): void;
+  on(event: 'error', listener: (e: Error) => void): void;
+
+  off(event: 'participantsJoined', listener: ParticipantJoinedListener): void;
+  off(event: 'participantsLeft', listener: ParticipantLeftListener): void;
+  off(event: 'meetingEnded', listener: ParticipantLeftListener): void;
+  off(event: 'error', listener: (e: Error) => void): void;
+
+  // Call interfaces needing re-declared
```

Comment:
Instead of redefining here, is there a clever way to do `Omit<CallAdapter, on(event: 'EVENT_TO_OMIT')>` instead of doing `Omit<CallAdapter, on(event: 'on')>` which I am doing currently